com.discordapp.Discord.appdata.xml

```diff
@@ -9,6 +9,10 @@
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>LicenseRef-proprietary</project_license>
   <url type="homepage">https://discord.com</url>
+  <url type="help">https://support.discord.com</url>
+  <url type="bugtracker">https://support.discord.com/hc/categories/115000168371</url>
+  <url type="contact">https://discord.com/company-information</url>
+  <url type="donation">https://discord.com/nitro</url>
```

Comment:
As you noted yourself, I don’t think calling this a donation page is accurate. Please remove it.

Reply:
Heh, done.